Dockerfile

```diff
@@ -15,5 +15,12 @@ RUN  \
      rm -rf /go/pkg /go/src /usr/local/go && apk del .build-deps
 
 EXPOSE 9000
-ENTRYPOINT ["minio"]
+
+COPY buildscripts/docker-entrypoint.sh /usr/bin/
+RUN ln -s usr/bin/docker-entrypoint.sh /entrypoint.sh && \
+     chmod +x /entrypoint.sh && \
+     chmod +x /usr/bin/docker-entrypoint.sh # backwards compat
+ENTRYPOINT ["docker-entrypoint.sh"]
+
 VOLUME ["/export"]
+CMD ["minio"]
```

Comment:
CMD is the default argument to entrypoint. Since entrypoint here is a script  `minio` becomes the default argument.